pkg/processor/reportprocessor.go

```diff
@@ -253,14 +253,27 @@ func getGovernanceResults(dynamicClient dynamic.Interface, clusterInfo types.Man
 				continue
 			}
 
+			timeString, _, err := unstructured.NestedString(historyItems[0].(map[string]interface{}), "lastTimestamp")
+			if err != nil {
+				glog.Warningf(
+					"error parsing compliance lastTimestamp as a string for policy %s, template %d: %s", plcName, idx, err)
+				continue
+			}
+			timestamp, err := time.Parse(time.RFC3339, timeString)
+			if err != nil {
+				glog.Warningf(
+					"error parsing compliance lastTimestamp into a time value for policy %s, template %d: %s", plcName, idx, err)
+				continue
+			}
+
 			// Append violation to policy report results
 			clusterViolations = append(clusterViolations, &v1alpha2.PolicyReportResult{
 				Policy:      plcName,
 				Description: message,
 				Scored:      false,
 				Category:    category,
 				Source:      "grc",
-				Timestamp:   metav1.Timestamp{Seconds: time.Now().Unix(), Nanos: int32(time.Now().UnixNano())},
+				Timestamp:   metav1.Timestamp{Seconds: timestamp.Unix(), Nanos: 0},
```

Comment:
Nit: Since the default value for an `int32` is 0, I think we can just omit it.
```suggestion
				Timestamp:   metav1.Timestamp{Seconds: timestamp.Unix()},
```

Reply:
accepted. Thanks